server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -1279,7 +1280,7 @@ public void setMailboxesShouldReturnNotUpdatedWhenNewParentDoesntExist() {
             .body(NAME, equalTo("mailboxesSet"))
             .body(ARGUMENTS + ".notUpdated", hasEntry(equalTo(mailboxId), Matchers.allOf(
                     hasEntry(equalTo("type"), equalTo("notFound")),
-                    hasEntry(equalTo("description"), equalTo("The parent mailbox 'badParent' was not found."))))); 
+                    hasEntry(equalTo("description"), containsString("badParent")))));
```

Comment:
Why can't it contain the whole error message ? Or at least a greater subset of it ?